sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -1206,40 +1194,26 @@ private boolean isManagementToken(String lockToken) {
         return managementNodeLocks.contains(lockToken);
     }
 
-    private Mono<Void> updateDisposition(MessageLockToken message, DispositionStatus dispositionStatus,
+    private Mono<Void> updateDisposition(String lockToken, DispositionStatus dispositionStatus,
         String deadLetterReason, String deadLetterErrorDescription, Map<String, Object> propertiesToModify,
         String sessionId, ServiceBusTransactionContext transactionContext) {
 
         if (isDisposed.get()) {
             return monoError(logger, new IllegalStateException(
                 String.format(INVALID_OPERATION_DISPOSED_RECEIVER, dispositionStatus.getValue())));
-        } else if (Objects.isNull(message)) {
-            return monoError(logger, new NullPointerException("'receivedMessage' cannot be null."));
-        } else if (Objects.isNull(message.getLockToken())) {
-            return monoError(logger, new NullPointerException("'receivedMessage.lockToken' cannot be null."));
-        } else if (message.getLockToken().isEmpty()) {
-            return monoError(logger, new IllegalArgumentException("'message.lockToken' cannot be empty."));
+        } else if (Objects.isNull(lockToken)) {
```

Comment:
The check doesn't match the documentation. This throws a NPE rather than IllegalArgumentExceptionwhen lockToken is null.

Reply:
Yes , I realize that. Thanks. I will change documentation.